Earthfile

```diff
@@ -114,10 +120,12 @@ lint-scripts-auth-test:
     # flag to source setup.sh during analysis.
     RUN shellcheck -x test-*.sh
 
+# lint-scripts runs the spellcheck package against both misc and test shell scripts
```

Comment:
I would change this comment to describe what higher-level task the target attempts to perform rather than describing the implementation details.

If someone really wanted to know what other targets it called, they could just simply look at the target's code block and see it just calls two helper targets.

```suggestion
# lint-scripts runs the shellcheck package to detect potential errors in shell scripts
```